src/server/api/routes/users/v2/nfts/featured.js

```diff
@@ -0,0 +1,77 @@
+const { Router } = require('express');
+const {
+  authenticateLogin,
+  checkWalletMatch,
+} = require('../../../../middleware/auth');
+const { setPrivateCacheHeader } = require('../../../../middleware/cache');
+const { handleRestfulError } = require('../../../../middleware/error');
+const { isValidAddress } = require('../../../../util/cosmos');
+const {
+  FieldValue,
+  walletUserCollection,
+} = require('../../../../../modules/firebase');
+
+const router = Router();
+
+router.get('/v2/users/:wallet/nfts/featured', async (req, res, next) => {
+  try {
+    const { wallet: user } = req.params;
+    if (!isValidAddress(user)) {
+      res.sendStatus(400);
+      return;
+    }
+    const userDoc = await walletUserCollection(user).get();
+    const { featuredNftClassIds = [] } = userDoc.data();
+    res.json({ featured: featuredNftClassIds });
```

Comment:
Prefer `NFT` over `Nft` in camelcase except for the first word
```suggestion
    const { featuredNFTClassIds = [] } = userDoc.data();
    res.json({ featured: featuredNFTClassIds });
```